test/test_conversion_driver_esm1p5.py

```diff
@@ -142,3 +165,71 @@ def test_convert_esm1p5_output_dir_error():
         esm1p5_convert.convert_esm1p5_output_dir(
             "/test_convert_esm1p5_output_dir_error/fake/path/"
         )
+
+
+def test_format_successes():
+    succeeded_inputs = [
+        Path("dir_1/fake_file_1"),
+        Path("./dir_2/fake_file_2"),
+        Path("/dir_3/fake_file_3")
+    ]
+    succeeded_outputs = [
+        Path("./fake_output_dir/file1.nc"),
+        Path("fake_dir_2/output_file.nc"),
+        Path("/dir_500/ncfile.nc")
+    ]
+
+    succeeded_pairs = list(zip(succeeded_inputs, succeeded_outputs))
+    # Perhaps don't need to convert to list as format_successes just needs 
+    # an iterable as input?
+
+    success_reports = esm1p5_convert.format_successes(succeeded_pairs)
+
+    # Check that the successful outputs make it into the report
+    for i, successful_output in enumerate(succeeded_outputs):
+        assert str(successful_output) in success_reports[i]
+
+
+def test_format_failures_quiet():
```

Comment:
Minor style thing: the func could be called `test_format_failures_quiet_mode()` for a touch more clarity.